### ISSUE TITLE
DOTA2：courier using equipment event

### ISSUE THREAD

**Issue by yangex** [skadistats/clarity#245]
i want to get information about courier equipment using，but through **@OnCombatLogEntry** ,`CombatLogEntry.getInflictorName().equals(”the equipment name(eg:item_smoke_of_deceit)”)`  can't get it.
I would appreciate it if you could help me.

**Comment by BusyLivingDota** [comment]
There is m_hTransientCastItem property in each CDOTA_Unit_Courier entity. But the property never perform properly. I asked my friend to use a smoke with courier at 31:13 in radiant fountain in game 6121405721 but its not detected. Maybe spheenik can you fix this property please? Just sayin.

**Comment by spheenik** [comment]
Sorry: I just provide the data in the replay, I do not modify it in any form. If the data sucks, you've got to complain to Gaben.

**Comment by BusyLivingDota** [comment]
Yeah I understand cant really blame spheenik for that. I use modifier_add combatlog in the current project and everything is ok except cant find smoke center when courier uses smk (but we know if there is no user for smoke, then user is a courier). It seems 
a smoke is the only item a courier can use. Luckily a pro tells me he didn't smoke in a courier for more than a year, so i guess no need to worry to much.
